bugbug/commit_features.py

```diff
@@ -136,54 +138,291 @@ class source_code_file_metrics(object):
+            "Maximum file number of source loc": commit["metrics"]["sloc_max"],
+            "Maximum file number of instruction loc": commit["metrics"]["ploc_max"],
+            "Maximum file number of logical loc": commit["metrics"]["lloc_max"],
+            "Maximum file number of comment loc": commit["metrics"]["cloc_max"],
+            "Maximum file number of function arguments": commit["metrics"]["nargs_max"],
+            "Maximum file number of function exit points": commit["metrics"][
+                "nexits_max"
+            ],
+            "Maximum file cognitive": commit["metrics"]["cognitive_max"],
+            "Minimum file cyclomatic": commit["metrics"]["cyclomatic_min"],
+            "Minimum file number of unique operands": commit["metrics"][
+                "halstead_n2_min"
+            ],
+            "Minimum file number of operands": commit["metrics"]["halstead_N2_min"],
+            "Minimum file number of unique operators": commit["metrics"][
+                "halstead_n1_min"
+            ],
+            "Minimum file number of operators": commit["metrics"]["halstead_N1_min"],
+            "Minimum file number of source loc": commit["metrics"]["sloc_min"],
+            "Minimum file number of instruction loc": commit["metrics"]["ploc_min"],
+            "Minimum file number of logical loc": commit["metrics"]["lloc_min"],
+            "Minimum file number of comment loc": commit["metrics"]["cloc_min"],
+            "Minimum file number of function arguments": commit["metrics"]["nargs_min"],
+            "Minimum file number of function exit points": commit["metrics"][
+                "nexits_min"
+            ],
+            "Minimum file cognitive": commit["metrics"]["cognitive_min"],
+            "Total file cyclomatic": commit["metrics"]["cyclomatic_total"],
+            "Total file number of unique operands": commit["metrics"][
+                "halstead_n2_total"
+            ],
+            "Total file number of operands": commit["metrics"]["halstead_N2_total"],
+            "Total file number of unique operators": commit["metrics"][
+                "halstead_n1_total"
+            ],
+            "Total file number of operators": commit["metrics"]["halstead_N1_total"],
+            "Total file number of source loc": commit["metrics"]["sloc_total"],
+            "Total file number of instruction loc": commit["metrics"]["ploc_total"],
+            "Total file number of logical loc": commit["metrics"]["lloc_total"],
+            "Total file number of comment loc": commit["metrics"]["cloc_total"],
+            "Total file number of function arguments": commit["metrics"]["nargs_total"],
+            "Total file number of function exit points": commit["metrics"][
+                "nexits_total"
+            ],
+            "Total file cognitive": commit["metrics"]["cognitive_total"],
+        }
+
+
+def merge_function_metrics(objects):
+    return {
```

Comment:
nit: create a temp `metrics = map(lambda o: o["metrics"], objects)`
nit: use a `map` instead of `for...`

Reply:
Fixed in another way by storing the list of metrics in the repository module and building the dict without having to specificy all the metrics